### ISSUE TITLE
Rev 2 isn't released yet

### ISSUE THREAD

**Issue by berzerk0** [berzerk0/Probable-Wordlists#24]
**TLDR;
I said Rev 2 would be out mid-july 2017. That's now.
New estimate is Mid-August**

I am not close to release. 

I had a major setback due to a minor script typo that required me to do about 30% of the total Rev 2 work all over again. *C'est la vie*, or perhaps *shikata ga nai* is better here. 

I'm pretty much back to where I was before the typo, if not farther along.
There's a lot of manual work that needs to be done that isn't script friendly. 
I see bash in my sleep, but luckily I am nearing the end of that portion.

Next up is the stage where I just set it up to run and go about my business.
New estimate is Mid-August.

**Comment by NoorMkdad** [comment]
It's OK my dude, doing this type of stuff for free is insane, keep up the good work :)

**Comment by berzerk0** [comment]
Rev 2 is delayed further. 
This time it's not due to accidental deletion on my part.

In the meantime, I will update the torrent and MegaLink files to include the updated WPA-Lists that have had 7char lines removed. I may also pre-release some Rev 2 content, such as updated Non-Password wordlists. 

Might be worthy of an intermediate release __1.3__

**Comment by spmedia** [comment]
+1 for an intermediate release 1.3! :)

**Comment by message** [comment]
Bump

**Comment by berzerk0** [comment]
Rev 2 is actually mostly done - I've just been studying for some certs and things that started to have more solid deadlines.

I'm still around!

**Comment by mdPlusPlus** [comment]
Do you think there will be a release before the turn of the year?

**Comment by NoorMkdad** [comment]
Has Rev 2 been released? It's 2078 already

**Comment by berzerk0** [comment]
In additional to juggling some family issues, I've reached a fork in the road

Original Plan:  Delay v2 further, but include a major feature.

New Plan:.  Release v2 sooner, but postpone release of this major feature. 

This major feature is country/language based wordlists. I have some concerns about this feature, however, so its future is in question.

**Comment by mdPlusPlus** [comment]
Any idea when the v2 release will probably take place?

**Comment by morangeman** [comment]
Thanks for the updates, looking forward to the new releases soon :)

**Comment by message** [comment]
@berzerk0, country based wordlist is cool feature, but it can wait. In Vietnam every 3rd password has a word _ngoc_ in it

**Comment by tuxayo** [comment]
Whatever happens, don't stress about this.
That's the real challenge: succeeding without paying a major personal cost.

Burning yourself out will prevent you moving this project forward and other amazing things in your life.
(I don't like a part of this argument (invoking the good of the project) but that might be the last one that works if others don't)

**Comment by ee1709** [comment]
You should consider _not_ removing passwords < 8 characters for the WPA password list, because people should be using some basic rules anyways (adding a digit), and you could be throwing away good passwords by pruning them.

**Comment by berzerk0** [comment]
@tuxayo thanks for the support! Right now the personal cost is patience as the desktop chugs along cranking out the project.

@ee1709 Let's move this to a different issue

**Comment by Ho52198** [comment]
First of all - Thank you. I first was shocked to find all these alphabetically or random wordlists. I know "probablility" is not something that you can just do, and is tricky, but fortunately, you use common sense :)

Second - any current alpha/beta/minor version will be welcome from the community, even if not covering everything. 

If you have anything usable on your PC - just give it as is :). People will be happy.

**Comment by berzerk0** [comment]
@Ho52198 - all I can provide right now is an update.

The method use makes it so nothing can be released until it is released all at once. All the files need to be found, standardized, and then combined together before they can be sorted by popularity.

"Standardization" has posed the biggest challenge. Some leaks only contain passwords, but files contain information that should not be released. That information needs to be removed. Duplicates need to be removed, newline characters are standardized, etc.


Here are the remaining steps (not necessarily in order):
* "Clean" all files - the files I will be using for Rev 2 have been standardized, so I just need to chop off non-pertinent information and remove duplicates
* Check for 1-to-1 identical files and remove them
* Combine all pure-password files into "The Blob"
* Sort "The Blob" by popularity - this involves producing a count of appearances next to each line
* Clone parts of "The Blob" and create the lists. 75+ appearances, 50+ appearances, etc.
* Create WPA Versions of those lists
* Compress Everything into 2 formats.
* Take down Revision 1, Upload Revision 2

**Comment by berzerk0** [comment]
![it's happening](https://pbs.twimg.com/media/DV80W3tW4AESwGt.jpg)

**Comment by Ho52198** [comment]
Thanks :)

**Comment by berzerk0** [comment]
https://pbs.twimg.com/media/DWDp0TjXUAAN2dY.jpg

Time to carve it up

**Comment by berzerk0** [comment]
If you're reading this, Release 2 is out.
